### PR TITLE
Add Logos Bible Software 6.14.0.65

### PR DESCRIPTION
**Pull request by ickc**
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/logos-bible-software.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'logos-bible-software' do
```
Comment (vitorgalvao) [inline Casks/logos-bible-software.rb]: The token reference was not followed. This needs to be `logos`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/logos-bible-software.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version '6.14.0.65'
```
Comment (vitorgalvao) [inline Casks/logos-bible-software.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/logos-bible-software.rb (hunk 1 above)
Comment (ickc) [inline Casks/logos-bible-software.rb]: Sorry I misunderstand it.

The one in the dmg is `Logos.app`, but the installed app is called `Logos Bible Software 6.app`. And I thought "logos" is too generic and people would thought it is the plural form of the English word logo, where there's another software named as logo which design logo. Rather, "logos" is a transliteration of "λόγος".

Anyway, I'm changing it to logos if you think that's right.

File: Casks/logos-bible-software.rb (hunk 2 above)
Comment (ickc) [inline Casks/logos-bible-software.rb]: Done and thanks

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**